app/frontend/src/types.ts

```diff
@@ -1,50 +1,64 @@
+// Represents a grounding file
 export type GroundingFile = {
     id: string;
     name: string;
     content: string;
 };
 
+// Represents an item in the history
 export type HistoryItem = {
     id: string;
     transcript: string;
     groundingFiles: GroundingFile[];
+    sender: "user" | "assistant"; // Add sender field
```

Comment:
nit: I'm not sure these comments are adding value, the property names and values already make them clear